div-calculator.rb

```diff
@@ -0,0 +1,75 @@
+# calculates answer after taking user input for operation to be 
+  
+# checks to make sure that symbol is valid
+def symbol_entry(symbol)
+  if symbol != "add" && symbol != "+" && symbol != "subtract" && symbol != "-" && symbol != "multiply" && symbol != "*" && symbol != "divide" && symbol != "/" && symbol != "^" && symbol != "%" && symbol != "exponent" && symbol != "module"
+    puts "Invalid symbol input!"
+  end 
+end
+ 
+# does the math according to numerical operator  
+def symbol_check(symbol, value1, value2) 
+print "Answer: " 
+  case symbol
+  when "add", "+"
+    print "#{value1} + #{value2} = " 
+    ans = value1 + value2
+  when "subtract", "-" 
+    print "#{value1} - #{value2} = "  
+    ans = value1 - value2
+  when "multiply" , "*"
+    print "#{value1} * #{value2} = " 
+    ans = value1 * value2
+  when "divide", "/"  
+    if value2 != 0
+      print"#{value1} / #{value2} = " 
+      ans = value1 / value2
+    else
+      puts "Infinity"
+      ans = 0
+    end
+  when "^", "exponent"
+    print"#{value1} ** #{value2} = " 
+    ans = value1 ** value2 
+  when "%" , "module"
+    print"#{value1} % #{value2} = " 
+    ans = value1 % value2 
+  else 
+    puts "Unable to calculate!"
+  end
+    
+  if ans % 1 > 0.0
```

Comment:
You get an error after the user gives you bad input, but only after your program prints "Unable to calculate!". See if after Thurday's lessons you can fix the error. If you have questions as to how you might go about this, feel free to talk to me about it.